values/prometheus-operator/rules/gitea-db-backup.yaml

```diff
@@ -0,0 +1,10 @@
+groups:
+  - name: gitea-db-backup
+    rules:
+      - alert: BackupGiteaDbFailing
+        expr: time() - cnpg_collector_last_failed_backup_timestamp{namespace="gitea",pod=~"gitea-db-1"} > 86400 or time() - cnpg_collector_last_failed_backup_timestamp{namespace="gitea",pod=~"gitea-db-2"} > 86400
```

Comment:
An extra thought (it does not need to be implemented in this PR). 
I have just realized that if you modify backup schedule to let's say every Friday, then 86400 may not fit. Should 86400 be configurable?

Reply:
Correct. For now I used the default 24 hours. Making this configurable requires some calculation logic based on the provided cron-type expression.